imageregistry/v1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
this is go 1.17+ new build constraint syntax btw, see https://pkg.go.dev/cmd/go#hdr-Build_constraints

Reply:
looks like CI didn't like it - removed.